engine/kube/kube.go

```diff
@@ -268,10 +281,14 @@ func (e *kubeEngine) Destroy(ctx context.Context, spec *engine.Spec) error {
 		),
 	)
 
-	// deleting the namespace should destroy all secrets,
-	// volumes, configuration files and more.
-	return e.client.CoreV1().Namespaces().Delete(
-		spec.Metadata.Namespace,
-		&metav1.DeleteOptions{},
-	)
+	if e.namespace == "" {
```

Comment:
`!=`?

Reply:
Ok, nvm. The code should be more more self-decriptive or documented in my opinion. Consider adding a bool `useSignleNamespace = e.namespace != ""` and rely on it in the checks.